doc/modules/ROOT/pages/vcdj.adoc

```diff
@@ -562,6 +568,37 @@ which the cue point was saved the value is `0000` and the CDJ displays
 “`00.0 Bars`”. On the next beat, the value becomes determined by the
 next cue point (if any) in the track.
 
+Byte `b7`, labeled _Mp_, seems to indicate local media presence status
+for players with multiple media slots. For players with only one slot,
+this value remains `0`.
+
+[[media-presence-flag-bits]]
+.Media presence flag bits.
+[bytefield]
+----
+(def boxes-per-row 8)
+(def box-width 70)
+(def left-margin 1)
+(draw-column-headers {:labels (str/split "7,6,5,4,3,2,1,0" #",")})
+(draw-box (hex-text 0 1))
+(draw-box (hex-text 0 1))
+(draw-box (hex-text 0 1))
+(draw-box (hex-text 0 1))
+(draw-box (hex-text 0 1))
+(draw-box (hex-text 0 1))
+(draw-box "USB")
+(draw-box "SD")
+----
+
+Byte `b8`, labeled _Ue_, indicates if USB media has been ejected.
+If media is inserted, this value returns to `0`.
+
+Byte `b9`, labeled _Se_, indicates if SD media has been ejected.
+If media is inserted, this value returns to `0`.
+
+Byte `ba`, labeled _el_, indicates if an emergency loop is currently
+active. Once a new track is loaded, this value returns to `0`.
```

Comment:
Presumably this will always be zero on CDJ-3000s with the firmware update that’s in the news this week, supporting full-track caching and thus the end of emergency loops! 😄

Reply:
I checked and this flag still flips to `1` on the CDJ-3000 with the new firmware. Even with full-track caching, the player seems to go into a half emergency mode, for example track skip buttons won't work and there is a yellow exclamation symbol on the screen :)

Reply:
Very interesting! I can merge this as it is, and then when I see the full text in context in a build of the document, perhaps just add a side note about the CDJ-3000 behavior.